<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.context.config.ContextNamespaceHandlerTests" tests="9" skipped="0" failures="0" errors="0" timestamp="2020-03-31T12:23:54" hostname="DESKTOP-LR4J8H3" time="0.21">
  <properties/>
  <testcase name="propertyPlaceholderLocationWithSystemPropertyForMultipleLocations()" classname="org.springframework.context.config.ContextNamespaceHandlerTests" time="0.028"/>
  <testcase name="propertyPlaceholderLocationWithSystemPropertyMissing()" classname="org.springframework.context.config.ContextNamespaceHandlerTests" time="0.016"/>
  <testcase name="propertyPlaceholderLocation()" classname="org.springframework.context.config.ContextNamespaceHandlerTests" time="0.03"/>
  <testcase name="propertyPlaceholderLocationWithSystemPropertyForOneLocation()" classname="org.springframework.context.config.ContextNamespaceHandlerTests" time="0.031"/>
  <testcase name="propertyPlaceholderIgnored()" classname="org.springframework.context.config.ContextNamespaceHandlerTests" time="0.018"/>
  <testcase name="propertyOverride()" classname="org.springframework.context.config.ContextNamespaceHandlerTests" time="0.026"/>
  <testcase name="propertyPlaceholderEnvironmentProperties()" classname="org.springframework.context.config.ContextNamespaceHandlerTests" time="0.014"/>
  <testcase name="propertyPlaceholderSystemProperties()" classname="org.springframework.context.config.ContextNamespaceHandlerTests" time="0.02"/>
  <testcase name="propertyPlaceholder()" classname="org.springframework.context.config.ContextNamespaceHandlerTests" time="0.023"/>
  <system-out><![CDATA[20:23:54.405 [Test worker] WARN  o.s.c.s.ClassPathXmlApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanDefinitionStoreException: Invalid bean definition with name 'foo' defined in class path resource [org/springframework/context/config/contextNamespaceHandlerTests-location-placeholder.xml]: Could not resolve placeholder 'foo' in value "${foo}"; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'foo' in value "${foo}"
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
